### PR TITLE
Add org.kde.konversation

### PR DESCRIPTION
**Pull request by grulja**
Konversation is a KDE IRC client application.

### CODE REVIEW

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "skip-appstream-check": true
```
Comment (bilelmoussaoui) [inline flathub.json]: We need to validate the appdata fo apps. This file should be removed

File: org.kde.konversation.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+    "id": "org.kde.konversation",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "default-branch": "stable",
```
Comment (bilelmoussaoui) [inline org.kde.konversation.json]:
```suggestion

```

File: org.kde.konversation.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "command": "konversation",
+    "rename-icon": "konversation",
+    "finish-args": [ "--device=dri",
+                     "--share=ipc",
+                     "--share=network",
+                     "--socket=x11",
+                     "--socket=wayland",
+                     "--filesystem=home",
```
Comment (bilelmoussaoui) [inline org.kde.konversation.json]: What do the applications need home access for?

**Comment by bilelmoussaoui** [comment]
bot, build org.kde.konversation

File: flathub.json (hunk 1 above)
Comment (grulja) [reply flathub.json]: I know, but given recent appdata requirements most of the KDE apps do not pass so until all the appdata are fixed in upstream, we skip the check.

Comment (nedrichards) [reply flathub.json]: I was happy to do this for existing apps in order to keep them updated, and also under the understanding that we were going to fix this upstream. For new apps I'd need to see a changeset/PR/similar to show that this is something that will be able to go away in a future release.

**Comment by grulja** [comment]
bot, build org.kde.konversation

File: flathub.json (hunk 1 above)
Comment (grulja) [reply flathub.json]: I pushed some appdata improvements, but unfortunately the test build doesn't seem to test that.

**Comment by grulja** [comment]
bot, build org.kde.konversation

File: flathub.json (hunk 1 above)
Comment (grulja) [reply flathub.json]: I did some appdata additions and it now seems to pass.

**Comment by nedrichards** [comment]
@grulja we need to send as many of these upstream as possible. I've been trying to get OARS in as a minimum but we need a better solution for release tags.

**Comment by nedrichards** [comment]
@grulja anyone else from upstream who should be added?

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.kde.konversation

**Comment by grulja** [comment]
> @grulja anyone else from upstream who should be added?

You can add @tsdgeos.

**Comment by tsdgeos** [comment]
:hand:

**Pull request closed without merging**